src/databricks/labs/remorph/helpers/db_workspace_utils.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+
+logger = logging.getLogger(__name__)
+
+
+class DatabricksSecretsClient:
```

Comment:
it looks like it's redundant because of https://github.com/databrickslabs/remorph/pull/254

Reply:
Refactored the `DatabricksSecretsClient` to `ReconConfigPrompts`, few functions are used in both the PR's. 
That could be the reason it's looking as redundant, but the PR's are segregated according to the functionality.